Spec.html

```diff
@@ -0,0 +1,252 @@
+<!doctype html>
+<meta charset="utf8">
+<link rel="stylesheet" href="ecmarkup.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/github.min.css">
+<script src="ecmarkup.js"></script>
+<pre class=metadata>
+title: Object Rest/Spread Properties
+stage: 2
+contributors: Sebastian Markbåge
+</pre>
+
+<emu-intro id="intro">
+	<h1>Introduction</h1>
+	<p>Rest properties collect the remaining own enumerable property keys that are not already picked off by the destructuring pattern. Those keys and their values are copied onto a new object.</p>
+
+	<pre><code class="javascript">
+    1: let { x, y, ...z } = { x: 1, y: 2, a: 3, b: 4 };
+    2: x; // 1
+    3: y; // 2
+    4: z; // { a: 3, b: 4 }
+	</code></pre>
+
+	<p>Spread properties in object initializers copies own enumerable properties from a provided object onto the newly created object.</p>
+
+	<pre><code class="javascript">
+    1: let n = { x, y, ...z };
+    2: n; // { x: 1, y: 2, a: 3, b: 4 }
+	</code></pre>
+</emu-intro>
+
+<emu-clause id="Spread">
+  <h1>Spread Properties</h1>
+
+  <h2>Syntax</h2>
+
+  <emu-production name="PropertyDefinition" params="Yield">
+    <emu-rhs><emu-nt params="?Yield">IdentifierReference</emu-nt></emu-rhs>
+    <emu-rhs><emu-nt params="?Yield">CoverInitializedName</emu-nt></emu-rhs>
+    <emu-rhs><emu-nt params="?Yield">PropertyName</emu-nt><emu-t>:</emu-t><emu-nt params="In, ?Yield">AssignmentExpression</emu-nt></emu-rhs>
+    <emu-rhs><emu-nt params="?Yield">MethodDefinition</emu-nt></emu-rhs>
+    <ins class="block">
+      <emu-rhs><emu-t>...</emu-t><emu-nt params="In, ?Yield">AssignmentExpression</emu-nt></emu-rhs>
```

Comment:
I added a [In, ?Yield] here to match the "PropertyName: AssignmentExpression" case above. Not sure if this is correct.